tests/aws.sh

```diff
@@ -125,50 +87,49 @@ INSTANCE_ID=$(aws ec2 run-instances \
 	)
 echo INSTANCE_ID=$INSTANCE_ID
 
-aws ec2 create-tags --resources $INSTANCE_ID \
-	--tags \
-	Key=Name,Value=rkt-tst-${DISTRO}-${GIT_BRANCH}-${FLAVOR} \
-	Key=Distro,Value=${DISTRO} \
-	Key=Repo,Value=${GIT_URL} \
-	Key=Branch,Value=${GIT_BRANCH} \
-	Key=Flavor,Value=${FLAVOR} \
-	Key=User,Value=${USER}
-
-while state=$(aws ec2 describe-instances \
+while state=$(aws --region $REGION ec2 describe-instances \
 	--instance-ids $INSTANCE_ID \
 	--output text \
 	--query 'Reservations[*].Instances[*].State.Name' \
 	); test "$state" = "pending"; do
   sleep 1; echo -n '.'
 done; echo " $state"
 
-AWS_IP=$(aws ec2 describe-instances \
+AWS_IP=$(aws --region $REGION ec2 describe-instances \
 	--instance-ids $INSTANCE_ID \
 	--output text \
 	--query 'Reservations[*].Instances[*].PublicIpAddress' \
 	)
 echo AWS_IP=$AWS_IP
 
-rm -f $CLOUDINIT
-
 sleep 5
-aws ec2 get-console-output --instance-id $INSTANCE_ID --output text |
+aws --region $REGION ec2 get-console-output --instance-id $INSTANCE_ID --output text |
   perl -ne 'print if /BEGIN SSH .* FINGERPRINTS/../END SSH .* FINGERPRINTS/'
```

Comment:
This perl thing to get the ssh fingerprint does not work. We could as well just delete it.